### PR TITLE
Add support to have shellscript workers part of pipeline too

### PR DESCRIPTION
**Pull request by tmathern**
## Description

- Handling some more cases for test mode
- Handle having 3d files as input (tooling used for images can not always handle all formats well, but it happens we don't need the extra metadata either)

Fixes #ASSETS-31792

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by badvision — APPROVED** [review]
Fair, though if there is a way in the future to extract metadata (unlikely though) we should revisit this.

File: lib/metadata.js [hunk 1]
```diff
@@ -249,10 +258,14 @@ class Metadata {
...
     }
-    
+
     async getImageMetadata(file) {
         let metadata = null;
-      
+
+        let fileExtension = path.extname(file);
+        fileExtension = fileExtension?.toLowerCase();
+        fileExtension = fileExtension?.slice(1);
```
Comment (pheenomenon) [inline lib/metadata.js:267]: we don't need this here ?

**Review by pheenomenon — CHANGES_REQUESTED** [review]
remove likely dead code ?

File: lib/metadata.js (hunk 1 above)
Comment (tmathern) [reply lib/metadata.js:267]: Good catch

**Comment by tmathern** [comment]
> remove likely dead code ?

Done!

**Review by pheenomenon — APPROVED** [review]

**Pull request merged**